Builder/Admin/ListBuilder.php

```diff
@@ -223,4 +229,45 @@ protected function findBatchActions()
             $this->addBatchAction($action);
         }
     }
+
+  /**
+   * Return a list of actions from excel.export
+   * 
+   * @return array
+   */
+  public function getExcelActions()
+  {
+      if (null === $this->excelActions) {
+          $this->excelActions = array();
+          $this->fillExportActions();
+      }
+
+      return $this->excelActions;
+  }
+
+  protected function fillExportActions()
+  {
+      $export = $this->getGenerator()->getFromYaml('builders.excel.params.export', []);
+      if (!count($export)) return;
+
+      foreach ($export as $keyName => $params ) {
+          if (!isset($params['show_button']) && (isset($params['show_button']) && filter_var($params['show_button'], FILTER_VALIDATE_BOOLEAN))) {
+              $action = new ExcelAction($keyName, $this);
```

Comment:
Due to the fact that you create new actions here, the credentials are not set and the check in the template does not work. The button is always printed...

Just add `$action->setCredentials($this->getExportParamsForKey($keyName, 'credentials', 'AdmingenAllowed'));`, that should do the trick.